bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/net/NetUtil.java

```diff
@@ -41,28 +49,52 @@
  * @author Markus Rathgeb - Initial contribution and API
  * @author Mark Herwege - Added methods to find broadcast address(es)
  * @author Stefan Triller - Converted to OSGi service with primary ipv4 conf
+ * @author Gary Tse - Network address change listener
  */
 @Component(configurationPid = "org.eclipse.smarthome.network", property = { "service.pid=org.eclipse.smarthome.network",
         "service.config.description.uri=system:network", "service.config.label=Network Settings",
-        "service.config.category=system" })
+        "service.config.category=system", "service.config.network.interface_poll.frequency_seconds=120" })
```

Comment:
What is this about?

Reply:
Configurable polling frequency; the original PR has a metatype xml file attached so that it can be changed in the UI too.  What do you think?

Reply:
I think it should be handled as an optional parameter, so there is no need to set it as a property here.